cluster-autoscaler/core/static_autoscaler_test.go

```diff
@@ -72,9 +74,24 @@ type daemonSetListerMock struct {
 	mock.Mock
 }
 
-func (l *daemonSetListerMock) List() ([]*extensionsv1.DaemonSet, error) {
+func (l *daemonSetListerMock) List(selector labels.Selector) ([]*appsv1.DaemonSet, error) {
```

Comment:
Just a general comment. We should not be writing mocks by hand. Mockery saves a lot of work. And generated mocks are actually more versatile. Could you log ticket to move to mockery?